lib/housing-available/housing-available.controllers.js

```diff
@@ -182,17 +182,26 @@ module.exports = (models) => {
       let housingAvailableInstance;
       try {
         housingAvailableInstance = await createHousingAvailable(housingAvailableObject);
-
       } catch (e) {
-        throw Boom.badImplementation("Housing available instance creation failed.");
+        throw Boom.badImplementation('Error during createHousingAvailable(housingAvailableObject): ' + e);
```

Comment:
please pass e as a second argument to .badImplementation. it should be `'...housingAvailableObject).', e`